robotools/evotools/__init__.py

```diff
@@ -309,6 +672,90 @@ def _partition_by_column(
     return column_groups
 
 
+def toHex(dec: int):
+    """Method from stackoverflow to convert decimal to hex."""
+    digits = "0123456789ABCDEF"
+    x = dec % 16
+    rest = dec // 16
+    if rest == 0:
+        return digits[x]
+    return toHex(rest) + digits[x]
```

Comment:
Ideally name it `to_hex` and put a link to the StackOverflow post into the docstring.
Otherwise, I think this is generically simple enough that you don't need to attribute the source, if you no longer have it.